cmd/transport-setup/commands/root.go

```diff
@@ -16,10 +16,14 @@ import (
 	"github.com/skycoin/skywire-services/pkg/transport-setup/config"
 )
 
-var configFile string
+var (
+	logLvl     string
+	configFile string
+)
 
 func init() {
 	rootCmd.Flags().StringVarP(&configFile, "config", "c", "", "path to config file\033[0m")
+	rootCmd.Flags().StringVarP(&logLvl, "loglvl", "l", "info", "set log level one of: info, error, warn, debug, trace, panic")
```

Comment:
We have `logLvl` here, but never set it as logging level.

Reply:
Good catch. Thanks